templates/model/spec.go

```diff
@@ -571,6 +577,42 @@ func (s *StringReplacement) UnmarshalYAML(n *yaml.Node) error {
 	return nil
 }
 
+// Append is an action that appends some output to the end of the file.
+type Append struct {
+	// Pos is the YAML file location where this object started.
+	Pos *ConfigPos `yaml:"-"`
+
+	Path              String `yaml:"path"`
+	With              String `yaml:"with"`
+	SkipEnsureNewline Bool   `yaml:"skip-ensure-newline"`
```

Comment:
The convention in spec.yaml is to use underscores rather than dashes. As in `regex_replace`, `strip_prefix`, etc

Reply:
`s/skip-ensure-newline/skip_ensure_newline/g`